docs/dapp/tools/remix.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Remix web IDE
+---
+
+# Remix
+
+## Intro
+
+[Remix] is a web-based Integrated Development Environment (IDE) designed for
+developing, testing, and deploying smart contracts on the Ethereum Network.
+This guide will show you how to use Remix in conjunction with MetaMask on the
+Sapphire Network.
+
+:::info
+
+For comprehensive details about Remix's features, consult the
+[Remix documentation].
+
+:::
+
+## Prerequisites
+
+1. Install the [MetaMask browser extension][metamask]
+2. Configure your networks:
+   - Add Sapphire networks to MetaMask using the `Add to MetaMask` buttons on
```

Comment:
(super nitpicky/not blocking) I wonder if a link back to the testnet faucet or mentioning that is the network used in the screenshot somehow could be helpful if someone came to this page first.